PyTorch/SpeechSynthesis/FastPitch/fastpitch/data_function.py

```diff
@@ -287,15 +286,15 @@ def get_prior(self, index, mel_len, text_len):
         return attn_prior
 
     def get_pitch(self, index, mel_len=None):
-        audiopath, *fields = self.audiopaths_and_text[index]
+        audiopath = self.audiopaths_and_text[index]['mels']
 
         if self.n_speakers > 1:
-            spk = int(fields[-1])
+            spk = int(self.audiopaths_and_text[index]['speaker'])
         else:
             spk = 0
 
         if self.load_pitch_from_disk:
-            pitchpath = fields[0]
+            pitchpath = self.audiopaths_and_text[index]['pitch']
             pitch = torch.load(pitchpath)
             if self.pitch_mean is not None:
                 assert self.pitch_std is not None
```

Comment:
commenting here because I can't do it lower down but what about the TTSCollate __call__ function?
I guess that's to do with the return type if __getitem__ returning a tuple?